modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -65,6 +65,8 @@ associateSyntaxInner(Syntax & syntax, ActionFactory & /*action_factory*/)
   registerSyntaxTask("DomainIntegralAction", "DomainIntegral", "add_vector_postprocessor");
   registerSyntaxTask("DomainIntegralAction", "DomainIntegral", "add_material");
 
+  registerSyntaxTask("CohesiveZoneModelAction", "CohesiveZoneModel", "add_interface_kernel");
```

Comment:
Follow the convention we are using with solid and line elements by nesting this so that the syntax looks like:
```
[Modules/TensorMechanics/CohesiveZoneMaster]
  [czm1]
  []
[]
```
Look at how LineElementMaster is done for an example.

Reply:
done